packages/core/src/index.js

```diff
@@ -330,23 +330,19 @@ const createCss = (init) => {
 	}
 
 	const css = (...inits) => {
-		let type
 		let composers = []
 		let composer
 		let defaultVariants = create(null)
 
 		for (const init of inits) {
 			if ($$composers in Object(init)) {
-				type = init.type || type
 				for (const composer of init[$$composers]) {
 					composers.push(composer)
 					assign(defaultVariants, composer.defaultVariants)
 				}
-			} else if (init && typeof init === 'object' && !('type' in init)) {
+			} else if (init && typeof init === 'object') {
```

Comment:
If `type` is pulled, this can become the `else` statement.

```suggestion
 			} else {
 ```